### PR TITLE
Fix M1 build

### PR DESCRIPTION
**Pull request by hjoaquim**
^

### CODE REVIEW

**Comment by tehcoderer** [comment]
Add this file to hooks 
https://github.com/OpenBB-finance/OpenBBTerminal/blob/6e6464dae545134a898e3848065f9d1794b3ebca/build/pyinstaller/hooks/hook-xgboost.py

File: build/pyinstaller/terminal.spec [hunk 1]
```diff
@@ -75,6 +75,10 @@ added_files = [
     (os.path.join(pathex, "blib2to3", "PatternGrammar.txt"), "blib2to3"),
 ]
 
+
+if is_darwin and os.path.exists(os.path.join(pathex, "xgboost")):
+    added_files.append((os.path.join(pathex, "xgboost"), "xgboost"))
```
Comment (hjoaquim) [inline build/pyinstaller/terminal.spec:80]: should the hook replace this? @tehcoderer

Comment (tehcoderer) [reply build/pyinstaller/terminal.spec:80]: yes sir you can remove that cause it' run with this
image

**Comment by jmaslek** [comment]
So I think this works?

**Comment by jmaslek** [comment]
So I think this works?

**Review by jmaslek — APPROVED** [review]

**Pull request merged**